tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -28,6 +28,27 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 		helper.CommonAfterEach(commonVar)
 	})
 
+	Context("user wants to work with services", func() {
+		When("user runs odo service commands with help flag", func() {
+			It("should display the help for odo service -h", func() {
+				appHelp := helper.Cmd("odo", "service", "-h").ShouldPass().Out()
+				Expect(appHelp).To(ContainSubstring("Perform service catalog operations"))
```

Comment:
IMO, we sholdn't be showing "service catalog" on CLI. I couldn't find another occurrence of this string being checked with a `Ctrl+F`. We should remove it, IMO.

Reply:
updated